polkadot/xcm/pallet-xcm-benchmarks/src/fungible/benchmarking.rs

```diff
@@ -46,8 +46,6 @@ benchmarks_instance_pallet! {
 		let asset = T::get_asset();
 
 		<AssetTransactorOf<T>>::deposit_asset(&asset, &sender_location, None).unwrap();
-		// check the assets of origin.
-		assert!(!T::TransactAsset::balance(&sender_account).is_zero());
```

Comment:
@franciscoaguirre I don't see it from PR, but did you remove all `T::TransactAsset`? If we don't need it, maybe we could remove it also or at least `where_clause { where` at the beginning?

Reply:
I left one where we test that we take delivery fees, which we are only using the native asset for now.